File: common/src/main/java/de/hype/bbsentials/environment/packetconfig/EnvironmentPacketConfig.java
package de.hype.bbsentials.environment.packetconfig;

import de.hype.bbsentials.client.common.client.BBsentials;

import java.awt.*;

public class EnvironmentPacketConfig {
    public static final String enviroment = "Client";
    public static final String notEnviroment = "Server";
    public static final int apiVersion = 1;
    public static Color getDefaultWaypointColor() {
        return BBsentials.visualConfig.waypointDefaultColor;
    }

    public static boolean getWaypointDefaultWithTracer() {
        return BBsentials.visualConfig.waypointDefaultWithTracer;
    }

    public static String getSelfUsername() {
        return BBsentials.generalConfig.getUsername();
    }
}

File: common/src/main/java/de/hype/bingonet/environment/packetconfig/EnvironmentPacketConfig.kt
package de.hype.bingonet.environment.packetconfig;

import de.hype.bingonet.client.common.client.BingoNet;

import java.awt.*;

public class EnvironmentPacketConfig {
    public static final String enviroment = "Client";
    public static final String notEnviroment = "Server";
    public static final int apiVersion = 1;
    public static Color getDefaultWaypointColor() {
        return BingoNet.visualConfig.waypointDefaultColor;
    }

    public static boolean getWaypointDefaultWithTracer() {
        return BingoNet.visualConfig.waypointDefaultWithTracer;
    }

    public static String getSelfUsername() {
        return BingoNet.generalConfig.getUsername();
    }
}
